examples/App.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Authzed API examples
+ */
+package demo;
+
+import com.authzed.grpcutil.ClientToken;
+import com.authzed.api.v0.ACLServiceGrpc;
+import com.authzed.api.v0.AclService;
+import com.authzed.api.v0.Core;
+import com.authzed.api.v1alpha1.Schema;
+import com.authzed.api.v1alpha1.SchemaServiceGrpc;
+import io.grpc.*;
+
+import java.util.concurrent.Executor;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+// Installation
+// https://search.maven.org/artifact/com.authzed.api/authzed
+
+public class App {
+    private static final Logger logger = Logger.getLogger(App.class.getName());
+    private static final String target = "grpc.authzed.com:443";
+    private static final String token = "tc_test_def_token";
+
+    private final SchemaServiceGrpc.SchemaServiceBlockingStub blockingStub;
+    private final ACLServiceGrpc.ACLServiceBlockingStub v0blockingStub;
+
+    public App(Channel channel) {
+        blockingStub = SchemaServiceGrpc.newBlockingStub(channel);
+        v0blockingStub = ACLServiceGrpc.newBlockingStub(channel);
+    }
+
+    public static void main(String[] args) {
+        ManagedChannel channel = ManagedChannelBuilder
+                .forTarget(target)
+                .useTransportSecurity()
+                .build();
+        try {
+            App client = new App(channel);
+//            client.write();
+//            client.read();
+//            client.relationship();
+            client.check();
+        } finally {
+            try {
+                channel.shutdownNow().awaitTermination(5, TimeUnit.SECONDS);
+            } catch (InterruptedException e) {
+                // Uh oh!
+            }
+        }
+    }
+
+    public String read() {
+        logger.info("Reading schema...");
+        Schema.ReadSchemaRequest request = Schema.ReadSchemaRequest
+                .newBuilder()
+                .addObjectDefinitionsNames("thelargeapp/article")
+                .build();
+        Schema.ReadSchemaResponse response;
+        try {
+            response = blockingStub
+                    .withCallCredentials(new ClientToken(token))
+                    .readSchema(request);
+        } catch (Exception e) {
+            logger.log(Level.WARNING, "RPC failed: {0}", e.getMessage());
+            return "";
+        }
+        logger.info("Greeting: " + response.toString());
+        return response.toString();
+    }
+
+    public String write() {
```

Comment:
can we call this "writeSchema"

Reply:
changed `read()` to `readSchema()` as well